src/Azure.Functions.Cli/Actions/AzureActions/LogStreamAction.cs

```diff
@@ -1,27 +1,53 @@
 ﻿using System;
 using System.IO;
 using System.Linq;
+using System.Net;
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Text;
 using System.Threading.Tasks;
 using Colors.Net;
-using Azure.Functions.Cli.Arm;
-using Azure.Functions.Cli.Helpers;
+using Fclp;
+using Azure.Functions.Cli.Arm.Models;
 using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Helpers;
+using static Azure.Functions.Cli.Common.OutputTheme;
 
 namespace Azure.Functions.Cli.Actions.AzureActions
 {
     [Action(Name = "logstream", Context = Context.Azure, SubContext = Context.FunctionApp, HelpText = "Show interactive streaming logs for an Azure-hosted Function App")]
     class LogStreamAction : BaseFunctionAppAction
     {
+        private const string ApplicationInsightsIKeySetting = "APPINSIGHTS_INSTRUMENTATIONKEY";
+        private const string LiveMetricsUriTemplate = "https://ms.portal.azure.com/#blade/AppInsightsExtension/QuickPulseBladeV2/ComponentId/{0}/ResourceId/{1}";
```

Comment:
```suggestion
        private const string LiveMetricsUriTemplate = "https://portal.azure.com/#blade/AppInsightsExtension/QuickPulseBladeV2/ComponentId/{0}/ResourceId/{1}";
```

Reply:
Sounds good. What's the difference btw? In my browser it always opened that. Is that internal?

Reply:
yes, if you log in using an @microsoft account, it redirects you to `ms.portal.azure.com` which is usually running a pre-release version of the UX.

Reply:
Ah, that makes sense. Thank you!